argo/settings.py

```diff
@@ -153,13 +153,11 @@
     "default": {
         "hosts": config.ELASTICSEARCH_HOSTS,
         "index": config.ELASTICSEARCH_INDEX,
-        "connection": config.ELASTICSEARCH_CONNECTION
+        "connection": config.ELASTICSEARCH_CONNECTION,
+        "api_key": config.get('ELASTICSEARCH_API_KEY')
```

Comment:
I am now questioning whether this will work. I think it might need to be:
```suggestion
        "api_key": getattr(config, 'ELASTICSEARCH_API_KEY')
```

Reply:
We can give it a try!